src/adapters/github.ts

```diff
@@ -309,7 +305,16 @@ class GithubAdapter extends GitAdapter {
   }
 
   protected getRepositoryUrl(repo: IRepo): string {
-    return `git@${gitHubEnterpriseBaseUrl}:${repo.owner}/${repo.name}.git`;
+    const gitHubEnterpriseBaseUrl = process.env.SHEPHERD_GITHUB_ENTERPRISE_BASE_URL ?? 'github.com';
```

Comment:
How moving the gitHubEnterpriseBaseUrl initialized outside class?

Reply:
As per my other comment, we should default to `api.github.com`.  Additionally, as this has already been rolled out to consumers, we would want to avoid breaking consumers that depend on it being defaulted to `api.github.com`.